src/main/resources/scriptDB/createDataBased.sql

```diff
@@ -0,0 +1,58 @@
+create schema if not exists travelagency collate utf8_general_ci;
+
+create table if not exists clients
+(
+	idClients int auto_increment
```

Comment:
Usually fields names are all in lowercase (name, surname, etc.)
Also it is good practice to name ID fields just "id" for better mapping in ORM frameworks